_posts/2025-03-17-release-0_11_0.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title:  "Kroxylicious release 0.11.0"
+date:   2025-03-17 00:00:00 +0000
+author: "Keith Wall"
+author_url: "https://github.com/k-wall"
+# noinspection YAMLSchemaValidation
+categories: blog kroxylicious releases
+tags: [ "releases", "kroxylicious" ]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.11.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.11.0). See the [Changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#0110) for a list of changes and summary of Deprecations, Changes and Removals.
+
+### Refreshed configuration file format
+
+The main change in this release is a revamped configuration file format.  We hope this change means that Kroxylicious is more intuative to configure.   As always, backward compatibility has been maintained with the old format, but you'll see deprecation
+warnings in the logs.  See the [changelog](https://github.com/kroxylicious/kroxylicious/blob/main/CHANGELOG.md#0110) for details of what you'll need to change.
```

Comment:
The changelog makes sense to us (because we did the work), but I don't think it's completely obvious to users what the main changes are. So users are going to have to follow all the links from the change logs to the PRs to figure out what's changed.
 
For this reason I think it might be nice to show how a simple-as-possible config looks before and after, and then refer to the changelog for the details.